src/index.scss

```diff
@@ -11,3 +11,16 @@ body {
   max-width: 500px;
   margin: 0 auto;
 }
+
+
+.circle5 {
+  display: inline-block;
+  padding: 6px 12px;
+  text-align: center;
+  color: #FFF;
+  background-color: #fb6b74;
+}
+h3{
+  color:#fb6b74;  font-size: 24px; border-bottom:1px solid #fb6b74;border-radius: 5px 5px 0px 0px;
```

Comment:
Please have a property in each line. Same reasons as stated above

Reply:
Done.